uwp/Charts/DataBinding.md

```diff
@@ -316,34 +316,73 @@ XBindingPath="ID" YBindingPath="Coal">
 
 {% highlight c# %}
 
-ScatterSeries series = new ScatterSeries()
-{
-
-    ItemsSource = new ViewModel().EnergyProductions,
-
-    XBindingPath = "ID",
-
-    YBindingPath = "Coal",
-
-    ScatterWidth = 20,
-
-    ScatterHeight = 20,
-
-    Label ="Coal",
 
-    ListenPropertyChange=true,
-
-    Interior = new SolidColorBrush(Color.FromRgb(0xBC, 0xBC, 0XBC))
-
-};
+      
+      public partial class MainWindow : Window
+      {
+     
+     
+            public MainWindow()
+            {
+                  InitializeComponent();
+            }
+
+            private void Button_Click_1(object sender, RoutedEventArgs e)
+            {
+                  // When you click on the button, the changes to XBindingPath and YBindingPath are updated in the output
+                  viewmodel.EnergyProductions[1].ID = "1001";
+                  viewmodel.EnergyProductions[1].Coal = 500;
+            }
+      }
+
+
+      public class Model : INotifyPropertyChanged
+      {
+            private string id;
+            public string ID
+            {
+                  get
+                  {
+                        return id;
+                  }
+                  set
+                  {
+                        id = value;
+                        OnPropertyChanged(nameof(ID));
+                  }
+            }
+
+            private string coal;
+            public string Coal
+            {
+                  get
+                  {
+                        return coal;
+                  }
+                  set
+                  {
+                        coal = value;
+                        OnPropertyChanged(nameof(Coal));
+                  }
+            }
+
+            public event PropertyChangedEventHandler PropertyChanged;
+
+            private void OnPropertyChanged(string propertyName)
+            {
+                  if (PropertyChanged != null)
+                  {
+                        PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
+                  }
+            }
+      }
 
-chart.Series.Add(series);
 
 {% endhighlight %}
 
 {% endtabs %}
 
-Also, When enabling this property to the series you need to implements INotifyPropertyChanged to the underlying data object to make the chart listen to the property changes of your data object.
+Also, When enabling this property to the series you need to implements INotifyPropertyChanged to the underlying data object of Model class to make the chart listen to the property changes of your data object.
```

Comment:
Also, when enabling this property of the series, you need to implement INotifyPropertyChanged in the underlying business model class to make the chart listen to the property changes of your data object.

Reply:
ok sir. I will change and update it